lib/fog/compute/ecloud/requests/ssh_key_create.rb

```diff
@@ -0,0 +1,76 @@
+module Fog
+  module Compute
+    class Ecloud
+      class Real
+        include Shared
+
+        def ssh_key_create(data)
+          validate_data([:name], data)
+
+          request(
+            :body => generate_ssh_key_create_request(data),
+            :expects => 201,
+            :method => "POST",
+            :headers => {},
+            :uri => data[:uri],
+            :parse => true
+          )
```

Comment:
Just wondering but wouldn't be better if the `request` method already returned the `body` so you won't have to keep (appending `body`) doing it all over again in every call?